zeppelin-server/src/test/java/org/apache/zeppelin/rest/NotebookRestApiTest.java

```diff
@@ -207,4 +210,51 @@ public void testClearAllParagraphOutput() throws IOException {
     //cleanup
     ZeppelinServer.notebook.removeNote(note.getId(), anonymous);
   }
+
+  @Test
+  @SuppressWarnings("unchecked")
+  public void testCreateNote() throws IOException {
+    // prepare data
+    Map<String, Object> data = Maps.newHashMap();
+    data.put("name", "test");
+    data.put("interpreter", "spark");
```

Comment:
perhaps we should test with a different interpreter? `spark` is likely the default even when this isn't set in the request